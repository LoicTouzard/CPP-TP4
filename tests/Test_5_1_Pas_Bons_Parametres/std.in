C cercle 0 0 1 1
EXIT
