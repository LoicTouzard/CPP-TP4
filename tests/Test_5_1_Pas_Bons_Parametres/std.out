ERR
#invalid parameters : 4 parameters expected
